redi-connect-front/src/components/molecules/FilterDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Checkbox, Icon } from '../atoms'
+import classnames from 'classnames'
+import './FilterDropdown.scss'
+
+interface Props {
+  label: string
+  selected: string[]
+  items: {label: string, value: string}[]
```

Comment:
Let's create a simple `type Item = {label: string, value: string}` above. We can then re-use down in the `items.map`.